.github/workflows/go.yml

```diff
@@ -0,0 +1,31 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code onto GOPATH
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+        path: /home/runner/work/sources-for-knative/sources-for-knative/go/src/github.com/vmware-tanzu/sources-for-knative
```

Comment:
I don't know enough about the syntax here but just curious why sources-for-knative is there twice?

Reply:
I can try without it, but as I was beating my head against the wall there was an error message about it not being under that directory (with it there twice) 🤣

Reply:
ok, it's running now.  If it blows up, I'll revert back to what was here.

Reply:
fails without it 🤷‍♂